src/main/java/frc/trigon/robot/commands/VisualizeNoteShootingCommand.java

```diff
@@ -82,12 +82,12 @@ private void configureStartingStats() {
         initialZVelocity = startingPitch.getSin() * startingTangentialVelocity;
     }
 
-    private boolean noteCrossedFieldWidth(double noteY) {
-        return noteY > FieldConstants.FIELD_WIDTH_METERS && !hasCrossedFieldWidthBorder;
+    private boolean noteCrossedCrossedAmpY(double noteY) {
```

Comment:
CrossedCrossed :)
This won't tell us if the note crossed, but if it's the first time it's past the ampy